sdk/core/azure-core/CHANGELOG.md

```diff
@@ -14,6 +14,7 @@
 ### Other changes and Improvements
 
 - Add high-level and simplified core.hpp file for simpler include experience for customers.
+- Update Exceptions as classes. 
```

Comment:
I wonder if this is considered a breaking change, in general.

```suggestion
- Update SDK-defined exception types to be classes instead of structs.
```